ui/components/connections/index.js

```diff
@@ -313,10 +320,14 @@ function Connections({
       name : "type",
       label : "Type",
       options : {
-        customHeadRender : function CustomHead({ index, ...column }) {
+        sort : true,
+        sortThirdClickReset : true,
+        customHeadRender : function CustomHead({ index, ...column }, sortColumn, columnMeta) {
           return (
-            <TableCell key={index}>
-              <b>{column.label}</b>
+            <TableCell key={index} onClick={() => sortColumn(index)}>
+              <TableSortLabel active={columnMeta.name === column.name} direction={columnMeta.direction || "asc"}>
+                <b>{column.label}</b>
+              </TableSortLabel>
```

Comment:
you can create a small reusable component `SortableTableCell` at top of file to remove the possible  duplication